bundles/CoreBundle/Templating/LegacyTemplateGuesser.php

```diff
@@ -47,10 +47,8 @@ public function __construct(KernelInterface $kernel, DelegatingEngine $templateE
      */
     public function guessTemplateName($controller, Request $request, $engine = 'twig')
     {
-        if ($request->attributes->has('_template')) {
-            if ($template = $request->attributes->get('_template') instanceof TemplatePhp) {
-                $engine = 'php';
-            }
+        if ($request->attributes->has('_template') && $request->attributes->get('_template') instanceof TemplatePhp) {
```

Comment:
Strictly speaking, the `$request->attributes->has('_template')` isn't needed here.

Reply:
go ahead! :)